pkgs/tools/system/skeema/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "skeema";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "skeema";
+    repo = "skeema";
+    rev = "v${version}";
+    sha256 = "1a75vixrpidim641809nj931r73zvbj2rsls7d80z7w87maav51m";
+  };
+
+  vendorSha256 = null;
+  doCheck = false;
```

Comment:
Why not running the tests?
Most of the tests pass and some don't because of NixOS specifics.
The simplest and cleanest way (if it's too difficult to patch) is to explicitly disable tests that fail for that reason.

Reply:
There are some integration tests which need environment setups. For now, I don't known how to skip specific tests. Maybe in the future per https://github.com/golang/go/issues/41583.

Reply:
Indeed, I didn't know it was not easy to skip tests with Go.

You could use `checkFlags = [ "-short" ];` anyway: it will skip a subset of the current failing tests :)